docs/installation/config-reference.rst

```diff
@@ -27,7 +27,7 @@ the ``zookeeper.properties`` file.  As an example, to set ``clientPort``, ``tick
     -e ZOOKEEPER_CLIENT_PORT=32181 \
     -e ZOOKEEPER_TICK_TIME=2000 \
     -e ZOOKEEPER_SYNC_LIMIT=2 \
-    confluentinc/cp-zookeeper:4.0.0
```

Comment:
to use the programatic versioning, define as: 

```
.. codewithvars::bash
 
     docker run -d \
     --net=host \
     --name=zookeeper \
     -e ZOOKEEPER_CLIENT_PORT=32181 \
     -e ZOOKEEPER_TICK_TIME=2000 \
     -e ZOOKEEPER_SYNC_LIMIT=2 \
     confluentinc/cp-zookeeper:|version|

```

Reply:
cc: @ewencp is that correct ^

Reply:
you want `|release|` instead of `|version|`. `release` is the full 3 digits, `version` is just the major.minor. These all come from variables in `conf.py` in docs, so you can use that file for reference if you're not sure which one you need, e.g. those are defined here: https://github.com/confluentinc/docs/blob/master/conf.py#L166-L169